plutus-core/plutus-core/src/PlutusCore/Default/Universe.hs

```diff
@@ -102,18 +103,18 @@ feature and have meta-constructors as built-in functions.
 -- See Note [Representing polymorphism].
 -- | The universe used by default.
 data DefaultUni a where
-    DefaultUniInteger              :: DefaultUni (Esc Integer)
-    DefaultUniByteString           :: DefaultUni (Esc BS.ByteString)
-    DefaultUniString               :: DefaultUni (Esc Text.Text)
-    DefaultUniUnit                 :: DefaultUni (Esc ())
-    DefaultUniBool                 :: DefaultUni (Esc Bool)
-    DefaultUniProtoList            :: DefaultUni (Esc [])
-    DefaultUniProtoPair            :: DefaultUni (Esc (,))
-    DefaultUniApply                :: !(DefaultUni (Esc f)) -> !(DefaultUni (Esc a)) -> DefaultUni (Esc (f a))
-    DefaultUniData                 :: DefaultUni (Esc Data)
+    DefaultUniInteger :: DefaultUni (Esc Integer)
+    DefaultUniByteString :: DefaultUni (Esc ByteString)
+    DefaultUniString :: DefaultUni (Esc Text)
```

Comment:
Got tired of maintaining alignment here.

Reply:
`M-x align-regexp` does the job, but I don't know if it's worth it.  Now that we have these very long type names you can end up with a lot of whitespace in the middle of a line if you align stuff.

Reply:
> `M-x align-regexp` does the job

Oh wow, thanks for telling me about that! I wish I knew about it earlier...

> Now that we have these very long type names you can end up with a lot of whitespace in the middle of a line if you align stuff.

Yeah, and `DefaultUniApply` no longer fits on a single line if we align stuff.